perfact/zodbsync/commands/with_lock.py

```diff
@@ -20,4 +18,4 @@ def add_args(parser):
 
     @SubCommand.with_lock
     def run(self):
-        subprocess.run(self.args.command, shell=True)
+        subprocess.check_call(self.args.command, shell=True)
```

Comment:
This changes the behavior, does it not? Can this be a problem, e.g., if commands regularly return non-zero exit codes?

Reply:
Well the only thing that happens after this call is the release of the lock, which also happens if the program crashes.

AFAIK no one uses this feature yet. I tried to use it once and found that it does not work with Python 2. In general I think raising an error is better that ignoring it. If the return code should be ignored, one can always use `|| true`.

Reply:
I thought as much, but I was guessing… thanks for clearing that up!

My review stands, then.